lib/uof_feed/messages/market.ex

```diff
@@ -0,0 +1,73 @@
+defmodule UofFeed.Messages.Market do
+  @moduledoc """
+  DataSchema representation of a Market.
+
+  ### Field list
+    - `id` - the ID of the market,
+    - `name` - the name of the market,
+    - `specifiers` - the market specifiers,
+    - `extended_specifers` - the market extended specifiers,
+    - `void_reason` - reason for voiding particular market at it's outcomes,
+    - `favourite` - recommended market, if present always set to 1,
+    - `status` - the current status of the market.
+    - `outcomes` - list of the `%Outcome{}` structs, describes possible market outcomes.
+
+  ### Void reason field mapping
+    - 0 - `OTHER`
+    - 1 - `NO_GOALSCORER`
+    - 2 - `CORRECT_SCORE_MISSING`
+    - 3 - `RESULT_UNVERIFIABLE`
+    - 4 - `FORMAT_CHANGE`
+    - 5 - `CANCELLED_EVENT`
+    - 6 - `MISSING_GOALSCORER`
+    - 7 - `MATCH_ENDED_IN_WALKOVER`
+    - 8 - `DEAD_HEAT`
+    - 9 - `RETIRED_OR_DEFAULTED`
+    - 10 - `EVENT_ABANDONED`
+    - 11 - `EVENT_POSTPONED`
+    - 12 - `INCORRECT_ODDS`
+    - 13 - `INCORRECT_STATISTICS`
+    - 14 - `NO_RESULT_ASSIGNABLE`
+    - 15 - `CLIENT_SIDE_SETTLEMENT_NEEDED`
+    - 16 - `STARTING_PITCHER_CHANGED`
```

Comment:
Same thing as change type field above, would be nice to have it mapped automatically to atoms.

Reply:
Same here, https://github.com/esl/uof_feed/issues/3 should cover this, too.